ui/shared/tabs/LogTab.jsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { LazyLog } from 'react-lazylog';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faExpand, faFileAlt } from '@fortawesome/free-solid-svg-icons';
+
+import TextLogStepModel from '../../models/textLogStep';
+import { getLogViewerUrl } from '../../helpers/url';
+
+const errorLinesCss = function errorLinesCss(logErrors) {
```

Comment:
I know this code is duplicated from the LogViewer `App`.  Once Suyash's PR is finalized and merged, I'll refactor this a bit to keep it `DRY`.

Reply:
This has now been rebased on Suyash's LogViewer PR.

Reply:
OK, I just added that function to the `display` helper.  Sorry to do this mid-stream while you're reviewing...